examples/demo3/demo3-scenario.yaml

```diff
@@ -0,0 +1,1073 @@
+version: 1.8.0
```

Comment:
I will provide an updated demo3-scenario.yaml as the current one had issues (PoA located in water, path crossing water, loops too short to see what is happening, 3 zones should have been 2, etc.)